Gemfile.lock

```diff
@@ -160,7 +160,9 @@ GEM
     mime-types (3.3.1)
       mime-types-data (~> 3.2015)
     mime-types-data (3.2019.1009)
-    mimemagic (0.3.5)
+    mimemagic (0.4.2)
+      nokogiri (~> 1)
+      rake (>= 0)
```

Comment:
Just FYI: my bundler changed this to
```
@@ -162,7 +162,7 @@ GEM
     mime-types-data (3.2019.1009)
     mimemagic (0.4.2)
       nokogiri (~> 1)
-      rake (>= 0)
+      rake
     mini_mime (1.0.2)
     mini_portile2 (2.5.0)
     mini_racer (0.2.9)
```

since >= 0 is vacuous. Doesn't affect functionality, but might be more semantic. Does yours explicitly give >= 0?

Reply:
Nope I agree >=0 is not needed. Removed in a new commit